src/main/java/mate/academy/Main.java

```diff
@@ -44,12 +57,35 @@
         yesterdayMovieSession.setMovie(fastAndFurious);
         yesterdayMovieSession.setShowTime(LocalDateTime.now().minusDays(1L));
 
-        MovieSessionService movieSessionService = null;
+        MovieSessionService movieSessionService =
+                (MovieSessionService) injector.getInstance(MovieSessionService.class);
         movieSessionService.add(tomorrowMovieSession);
         movieSessionService.add(yesterdayMovieSession);
 
         System.out.println(movieSessionService.get(yesterdayMovieSession.getId()));
         System.out.println(movieSessionService.findAvailableSessions(
-                        fastAndFurious.getId(), LocalDate.now()));
+                fastAndFurious.getId(), LocalDate.now()));
+
+        UserService userService = (UserService)
+                injector.getInstance(UserService.class);
+        User user = new User();
+        user.setEmail("bob@gmail.com");
+        user.setPassword("12345");
+        userService.add(user);
+
+        TicketDao ticketDao = (TicketDao)
+                injector.getInstance(TicketDao.class);
+        Ticket ticket = new Ticket();
+        ticket.setMovieSession(yesterdayMovieSession);
+        ticket.setUser(user);
+        ticketDao.add(ticket);
+
+        Order order = new Order();
+        final OrderDao orderDao = (OrderDao)
+                injector.getInstance(OrderDao.class);
+        order.setTickets(List.of(ticket));
+        order.setUser(user);
+        order.setLocalDateTime(LocalDateTime.now());
+        orderDao.add(order);
```

Comment:
Instead of directly using `OrderDao`, consider using `OrderService` to manage orders. This aligns with the service-oriented architecture and ensures that business logic is handled within the service layer.